cluster/manifests/kube-janitor/rules-config.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Should also only be deployed to non-production environments IMO.

Reply:
done